src/sources/eventstoredb_metrics/mod.rs

```diff
@@ -154,7 +154,7 @@ mod integration_tests {
     use super::*;
     use crate::test_util::components::{run_and_assert_source_compliance, SOURCE_TAGS};
 
-    const EVENTSTOREDB_SCRAPE_ADDRESS: &str = "http://localhost:2113/stats";
+    const EVENTSTOREDB_SCRAPE_ADDRESS: &str = "http://eventstoredb:2113/stats";
```

Comment:
**nitpick (non-blocking)**: this should follow the same pattern of ENV VAR unwrap or default value.